Project/Front-End/career-connect/src/components/Homepage.jsx

```diff
@@ -1,23 +1,55 @@
 import { useNavigate } from "react-router-dom";
 import JobCard from "./JobCard";
+import { useState, useEffect } from "react";
+import axios from "axios";
 
 
 export default function HomePage() {
     const navigate = useNavigate();
-
+    const [jobs, setjobs] = useState("")
+    const [location, setlocation] = useState("")
+    const [inputvalue, setinputvalue] = useState("")
+    const onchangehandler  = (event) => {
```

Comment:
This is likely a behavior that can be specify within the inputs `onChange` field but not a big deal to have it here